### DISCUSSION TITLE
Convert - Ignore Error "length and filesize do not match"

### DISCUSSION THREAD

**Discussion by medianotion** [dlemstra/Magick.NET#1455] [Help] [answered]
I have some images that convert successfully with IM CLI but do report "length and filesize do not match error/bmp.c/ReadBMPImage/934'" error.  The error causes Magick.Net to exit.  Apparently, the error is not fatal for IM CLI.  I was wondering if there was a way to ignore non-fatal errors in Magick.Net.  Please let me know.

**Comment by dlemstra** [comment, marked as answer]
You can ignore this if you use the `BmpReadDefines` when reading the image.
